### PR TITLE
Feat: added content to history page

### PR DESCRIPTION
**Pull request by Pr0bka**
anton lox

### CODE REVIEW

**Comment by AntonOvc** [comment]
I propose to add slight green/red background for incoming/outcoming transactions respectively or highlight the amount with red/green to indicate transaction state

File: src/components/Transactions/TransactionItem.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import classes from './TransactionItem.module.css';
+
+const TransactionItem = (props) => {
+  const formatDate = (arg) => {
+    const readableDate = new Date(arg);
+    let day = readableDate.getDate();
+    let month = readableDate.getMonth() + 1;
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionItem.js:7]: why let? prefer const

File: src/components/Transactions/TransactionItem.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  return (
+    <li className={classes.transaction}>
+      <div>
+        <h3 className={classes.transactionHeader}>Транзакция</h3>
+        <div className={classes.description}>Отправитель: {props.sender}</div>
+        <div className={classes.description}>Получатель: {props.receiver}</div>
+        <div className={classes.description}>Описание: {props.description}</div>
+        <div className={classes.description}>
+          Дата: {JSON.stringify(formatDate(props.date))}
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionItem.js:26]: please use props destructuring

File: src/components/Transactions/TransactionItem.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import classes from './TransactionItem.module.css';
+
+const TransactionItem = (props) => {
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionItem.js:3]: ({
sender,
receiver,
...etc
})

File: src/components/Transactions/TransactionsList.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import Spinner from '../Spinner/Spinner';
+const TransactionsList = () => {
+  const [transactions, setTransactions] = useState([]);
+  const [users, setUsers] = useState([]);
+  const { getTransactions, getUsers, loading } = useHomeApi();
+  useEffect(() => {
+    getTransactions().then(setTransactions);
+  }, []);
+
+  useEffect(() => {
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionsList.js:14]: move to the first useEffect

File: src/components/Transactions/TransactionsList.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    getUsers().then(setUsers);
+  }, []);
+
+  if (loading === true) {
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionsList.js:18]:
```suggestion
  if (loading) {
```

File: src/components/Transactions/TransactionsList.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      return (<Spinner/>)
+    } else if (loading === false && transactions.length === 0) {
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionsList.js:20]:
```suggestion
    } else if (!loading && transactions.length === 0) {
```

File: src/components/Transactions/TransactionsList.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      return <p className={classes.noTransactionsAvailable}>Суй ананас в жопу</p>;
+  }
+
+  const formatUserName = (arg) => {
```
Comment (AntonOvc) [inline src/components/Transactions/TransactionsList.js:24]: get rid of `arg` please

File: src/components/Transactions/TransactionItem.js (hunk 1 above)
Comment (Pr0bka) [reply src/components/Transactions/TransactionItem.js:17]: you are wrong, these variables are changing

File: src/components/Transactions/TransactionItem.js (hunk 2 above)
Comment (Pr0bka) [reply src/components/Transactions/TransactionItem.js]: fixed

File: src/components/Transactions/TransactionsList.js (hunk 6 above)
Comment (Pr0bka) [reply src/components/Transactions/TransactionsList.js]: fixed

**Review by AntonOvc — APPROVED** [review]

**Pull request merged**